docs/hugo/content/introduction/authentication.md

```diff
@@ -3,10 +3,11 @@ title: Authentication in Azure Service Operator v2
 linktitle: Authentication
 ---
 
-Azure Service Operator supports two different styles of authentication today.
+Azure Service Operator supports three different styles of authentication today.
```

Comment:
Are these listed in order of preference or maybe in order of recommendation?

Reply:
They're in order of recommendation _kinda_.

SP is first because it's the simplest/easiest to understand and get started with.
Workload identity is second because it's a better (but more complicated) way to set things up.
Pod Identity is third because it's deprecated and has been replaced by workload identity, but still works and we may still have customers using it.

I'll add a note to the Pod identity options that explicitly calls this out.

Reply:
Added these notes